storm-webapp/src/main/java/org/apache/storm/daemon/ui/filters/AuthorizedUserFilter.java

```diff
@@ -79,7 +80,7 @@
     public static Response makeResponse(Exception ex, ContainerRequestContext request, int statusCode) {
         String callback = null;
 
-        if (request.getMediaType().toString().contains("application/json")) {
+        if (request.getMediaType() != null && request.getMediaType().equals(MediaType.APPLICATION_JSON_TYPE)) {
```

Comment:
Nit: I think this can be simplified to `MediaType.APPLICATION_JSON_TYPE.equals(request.getMediaType())`